packages/s3-store/index.ts

```diff
@@ -233,6 +278,61 @@ export class S3Store extends DataStore {
     return data.ETag as string
   }
 
+  private async downloadIncompletePart(id: string) {
+    const incompletePart = await this.getIncompletePart(id)
+
+    if (!incompletePart) {
+      return
+    }
+    const filePath = await this.uniqueTmpFileName('tus-s3-incomplete-part-')
+
+    try {
+      let incompletePartSize = 0
+
+      const byteCounterTransform = new stream.Transform({
```

Comment:
Using a transformer to obtain the incomplete part's size appear a bit complex to me. Two alternatives are either using the length that is returned in the `GetObject` call to S3 (AFAIK there should be a length in there) or reading the file size after the part has been written to disk. Then we can remove this additional stream.

Reply:
I just saw that this code was copied from somewhere else. Feel free to ignore it then :)